src/constants.json

```diff
@@ -5,6 +5,6 @@
   "resolvers": {
     "avatar": ["lens", "ens", "snapshot"],
     "token": ["trustwallet", "zapper"],
-    "space": ["space"]
+    "space": ["space", "spacesx"]
```

Comment:
I'm not sure if it makes sense to have them as same entity, if we want to keep it the same I'd suggest reversing the order, because this impacts priority (last element overrides previous ones).

I think it would make sense to create separate resolver for `space-sx` (will need to update UI then).

What do you think @bonustrack ?

Reply:
Yes agree it should be a separate resolver here, we don't want load image on both S and SX for the same request.

Reply:
https://github.com/snapshot-labs/sx-ui/pull/576